order-service/src/main/java/io/eventuate/examples/tram/ordersandcustomers/orders/web/OrderController.java

```diff
@@ -44,6 +53,11 @@ public CreateOrderResponse createOrder(@RequestBody CreateOrderRequest createOrd
      return makeSuccessfulResponse(order);
   }
 
+  @RequestMapping(value = "/orders/make-snapshot", method = RequestMethod.POST)
+  public List<TopicPartitionOffset> makeSnapshot() {
```

Comment:
I think that it should return a JSON object rather than a List - that's always more extra work to process.

Reply:
I will fix